.vscode/launch.json

```diff
@@ -12,15 +12,15 @@
                 "miDebuggerPath": "${env:ARM_GCC_PATH}/arm-none-eabi-gdb.exe",
             },
             "targetArchitecture": "arm",
-            "program": "${workspaceRoot}/build/main.elf",
+            "program": "${workspaceRoot}/build/stm32_project_template.elf",
             "setupCommands": [{
-                    "text": "file '${workspaceRoot}/build/main.elf'"
+                    "text": "file '${workspaceRoot}/build/stm32_project_template.elf'"
                 },
                 {
                     "text": "target extended-remote localhost:2331"
```

Comment:
2331 não funciona, o 4242 sim

Reply:
? Mas isso é uma porta genérica do PC, o importante é estar igual a dali de baixo \/

Reply:
Se o padrão do st-util for 4242, aí acho válido mudar aqui pra deixar o padrão sim

Reply:
Pela help do st-util, a porta padrão é 4242 mesmo.